src/chainstate/coordinator/mod.rs

```diff
@@ -683,6 +688,7 @@ impl<'a, T: BlockEventDispatcher, N: CoordinatorNotices, U: RewardSetProvider>
                                 transaction.payload
                             {
                                 let contract_id = contract_call.to_clarity_contract_id();
+                                increment_stx_smart_contracts(); //promserver
```

Comment:
This metric is more accurately "contract calls processed" , rather than "stx_smart_contracts".

Reply:
renamed to `increment_contract_calls_processed`